<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GymSync</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(220, 220, 220);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <!-- Login Page -->
      <widget class="QWidget" name="login_page">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="logo_label">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>logo.png</pixmap>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="email_icon">
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>40</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap>email_icon.png</pixmap>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="email_input">
              <property name="placeholderText">
               <string>Correo</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="password_icon">
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>40</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap>password_icon.png</pixmap>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="password_input">
              <property name="echoMode">
               <enum>QLineEdit::Password</enum>
              </property>
              <property name="placeholderText">
               <string>Contraseña</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="login_error">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 0, 0);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="login_button">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(150, 150, 150);</string>
            </property>
            <property name="text">
             <string>Iniciar sesión</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="register_button">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(150, 150, 150);</string>
            </property>
            <property name="text">
             <string>Registrarse</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>

      <!-- Main Page -->
      <widget class="QWidget" name="main_page">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="day_banner">
          <property name="text">
           <string>RUTINA DE ENTRENAMIENTO</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="monday_label">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(180, 180, 180);</string>
            </property>
            <property name="text">
             <string>Monday
ARM DAY</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="tuesday_label">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(180, 180, 180);</string>
            </property>
            <property name="text">
             <string>Tuesday
PUSH DAY</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="wednesday_label">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(180, 180, 180);</string>
            </property>
            <property name="text">
             <string>Wednesday
LEGS DAY</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QScrollArea" name="exercise_scroll_area">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="exercise_container">
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QLabel" name="exercise1">
              <property name="text">
               <string>EJERCICIO 1</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="exercise2">
              <property name="text">
               <string>EJERCICIO 2</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="exercise3">
              <property name="text">
               <string>EJERCICIO 3</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="exercise4">
              <property name="text">
               <string>EJERCICIO 4</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="exercise5">
              <property name="text">
               <string>EJERCICIO 5</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="profile_button">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(150, 150, 150);</string>
            </property>
            <property name="text">
             <string>TU PERFIL</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="progress_button">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(150, 150, 150);</string>
            </property>
            <property name="text">
             <string>PROGRESO</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="notifications_button">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(150, 150, 150);</string>
            </property>
            <property name="text">
             <string>NOTIFICACIONES</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="user_email_label">
          <property name="text">
           <string>user@gmail.com</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>

      <!-- Profile Page -->
      <widget class="QWidget" name="profile_page">
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="profile_title">
          <property name="text">
           <string>DATOS PERSONALES</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="name_label">
            <property name="text">
             <string>Nombre y apellidos</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="name_input"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="profile_email_label">
            <property name="text">
             <string>Correo</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="profile_email_input"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="profile_password_label">
            <property name="text">
             <string>Contraseña</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="profile_password_input">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="confirm_password_label">
            <property name="text">
             <string>Confirmar contraseña</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="confirm_password_input">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="phone_label">
            <property name="text">
             <string>Número de teléfono</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="phone_input"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="age_label">
            <property name="text">
             <string>Edad</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="age_input"/>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="gender_label">
            <property name="text">
             <string>Género</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QComboBox" name="gender_combo">
            <item>
             <property name="text">
              <string>Masculino</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Femenino</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Otro</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="weight_label">
            <property name="text">
             <string>Peso (kg)</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QLineEdit" name="weight_input"/>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="height_label">
            <property name="text">
             <string>Altura (cm)</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QLineEdit" name="height_input"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="objective_label">
          <property name="text">
           <string>OBJETIVO</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="objective_combo">
          <item>
           <property name="text">
            <string>Perder peso</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Ganar masa muscular</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Mantenimiento</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="availability_label">
          <property name="text">
           <string>DISPONIBILIDAD DE HORARIO</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="availability_combo">
          <item>
           <property name="text">
            <string>Mañana</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Tarde</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Noche</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lifestyle_label">
          <property name="text">
           <string>ESTILO DE VIDA</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="lifestyle_combo">
          <item>
           <property name="text">
            <string>Sedentario</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Moderadamente activo</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Muy activo</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QPushButton" name="save_profile_button">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(150, 150, 150);</string>
            </property>
            <property name="text">
             <string>GUARDAR</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="cancel_profile_button">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(150, 150, 150);</string>
            </property>
            <property name="text">
             <string>SALIR SIN GUARDAR</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>

      <!-- Register Page -->
      <widget class="QWidget" name="register_page">
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QLabel" name="register_title">
          <property name="text">
           <string>DATOS PERSONALES</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="reg_name_label">
            <property name="text">
             <string>Nombre y apellidos</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="reg_name_input"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="reg_email_label">
            <property name="text">
             <string>Correo</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="reg_email_input"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="reg_password_label">
            <property name="text">
             <string>Contraseña</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="reg_password_input">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="reg_confirm_password_label">
            <property name="text">
             <string>Confirmar contraseña</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="reg_confirm_password_input">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="objective_label_2">
          <property name="text">
           <string>OBJETIVO</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="reg_objective_combo">
          <item>
           <property name="text">
            <string>Perder peso</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Ganar masa muscular</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Mantenimiento</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="availability_label_2">
          <property name="text">
           <string>DISPONIBILIDAD DE HORARIO</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="reg_availability_combo">
          <item>
           <property name="text">
            <string>Mañana</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Tarde</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Noche</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="save_register_button">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(150, 150, 150);</string>
          </property>
          <property name="text">
           <string>GUARDAR</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>

      <!-- Progress Page -->
      <widget class="QWidget" name="progress_page">
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QLabel" name="progress_title">
          <property name="text">
           <string>PROGRESO PERSONAL</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTabWidget" name="progress_tabs">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="weekly_tab">
           <attribute name="title">
            <string>Progreso semanal</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <item>
             <widget class="QLabel" name="weekly_chart_placeholder">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>200</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string>Gráfico de progreso semanal</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTableWidget" name="attendance_table">
              <property name="rowCount">
               <number>4</number>
              </property>
              <property name="columnCount">
               <number>7</number>
              </property>
              <row/>
              <row/>
              <row/>
              <row/>
              <column>
               <property name="text">
                <string>Lun</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Mar</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Mié</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Jue</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Vie</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Sáb</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Dom</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="monthly_tab">
           <attribute name="title">
            <string>Progreso diario</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_10">
            <item>
             <widget class="QLabel" name="monthly_chart_placeholder">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>300</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string>Gráfico de progreso diario</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="metric1_label">
            <property name="text">
             <string>Tiempo de actividad</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="metric2_label">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 0, 0);</string>
            </property>
            <property name="text">
             <string>Kcal</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="back_to_main_button">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(150, 150, 150);</string>
          </property>
          <property name="text">
           <string>VOLVER</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>